app/views/layouts/base.html.erb

```diff
@@ -3,6 +3,7 @@
 
 <head>
   <meta charset="utf-8" />
+  <meta name="theme-color" content="#112e51">
```

Comment:
Moving this up as the browser can set the color before any blocking JS/CSS scripts load.